src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -175,13 +192,16 @@
                  [:li {} (str subs " " status)])])]))]]
      (style/create-section-header "Description")
      (style/create-paragraph
-       (let [description (not-empty (get-in ws [:workspace :description]))]
-         (cond (:editing? @state) (react/create-element [MarkdownEditor {:ref "description" :initial-text description}])
+       (let [description (not-empty description)]
+         (cond editing? (react/create-element [MarkdownEditor {:ref "description" :initial-text description}])
                description [MarkdownView {:text description}]
                :else [:span {:style {:fontStyle "italic"}} "No description provided"])))
-     (attributes/view-attributes state refs)
+     [attributes/WorkspaceAttributeViewerEditor {:ref "workspace-attribute-editor"
+                                                 :editing? editing?
+                                                 :workspace-attributes workspace-attributes
+                                                 :workspace-bucket bucketName}]
      ;; TODO commented out until ready
-     ;[library/LibraryAttributeViewer {:library-attributes (not-empty (get-in ws [:workspace :library-attributes]))
```

Comment:
Let's talk offline about feature flags. It sounds like we want to have that capability across our codebases.